packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -39,7 +39,15 @@ function connect() {
           level: 'debug'
         },
         meetings: {
-          deviceType: 'WEB'
+          deviceType: 'WEB',
+          reconnection: {
+            enabled: true,
+            detection: true,
+            // Timeout duration to wait for ICE to reconnect if a disconnect is received.
+            iceReconnectionTimeout: 10000,
+            // Amount of times attempting to rejoin a meeting during reconnect
+            maxRejoinAttempts: 3
```

Comment:
Since these three are defaults, we don't need to set them, just the enabled flag.

Reply:
done